mypy/fastparse.py

```diff
@@ -769,6 +769,11 @@ def visit_Str(self, n: ast35.Str) -> Union[UnicodeExpr, StrExpr]:
         else:
             return UnicodeExpr(n.s)
 
+    # JoinedStr(expr* values)
```

Comment:
This whole method should be inside something like `if hasattr(ast35, 'JoinedStr'):` so that mypy will still work with the previous version of typed_ast.